anvil/src/x11.rs

```diff
@@ -328,10 +322,9 @@ pub fn run_x11(log: Logger) {
             window.set_cursor_visible(cursor_visible);
 
             // Send frame events so that client start drawing their next frame
-            state
-                .window_map
-                .borrow()
-                .send_frames(start_time.elapsed().as_millis() as u32);
+            window_map.send_frames(start_time.elapsed().as_millis() as u32);
+            std::mem::drop(window_map);
```

Comment:
This seems a bit odd to put in the busy loop?

Reply:
It is necessary for the next line, because otherwise state is borrowed.

Reply:
Oh it's probably because window map is in a ref cell

Reply:
Oh it's probably because window map is in a ref cell